src/LiveComponent/composer.json

```diff
@@ -29,7 +29,7 @@
         "php": ">=8.1",
         "symfony/property-access": "^5.4.5|^6.0|^7.0",
         "symfony/ux-twig-component": "^2.8",
-        "twig/twig": "^2.14.7|^3.0.4"
+        "twig/twig": "^2.14.7|~3.8.0"
```

Comment:
Minor - but technically we are still ok if you’re using 3.0.4. We just want to not allow 3.9. Perhaps we should write it to still allow the lower twig 3 versions?

Reply:
How would you write this ? If we use `<=` it will allow twig 1.0 no ?